src/main/scala/org/apache/spark/shuffle/ucx/perf/UcxPerfBenchmark.scala

```diff
@@ -107,13 +107,15 @@ object UcxPerfBenchmark extends App with Logging {
     val blocks = Array.ofDim[BlockId](options.numOutstanding)
     val callbacks = Array.ofDim[OperationCallback](options.numOutstanding)
     val requestInFlight = new AtomicInteger(0)
+    val rnd = new scala.util.Random
+    val blocksPerFile = options.numBlocks / options.files.length
```

Comment:
the actual numBlocks is blocksPerfFile * options.files.length in case options.numBlocks > blocksPerfFile * options.files.length, is that expected?

Reply:
So the idea is the following: you specify total number of blocks (`-n` parameter) and files. So each file will contain `numBlocks/numFiles`. E.g. total number of blocks in 1000 of 1M, and 10 files - so each file will have 100 blocks of 1M

Reply:
i meant for example numBlocks = 1000 and 8 files, will each file have 100 blocks and only 800 blocks in total actually be used?

Reply:
what if numBlocks = 1000 and 9 files, will each file have 111 blocks and only 999 blocks in total actually be used?